### DISCUSSION TITLE
[Feature Request] Remove alt+space shortcut

### DISCUSSION THREAD

**Discussion by raspin0** [linuxdeepin/developer-center#2839] [Features Request & Ideas | 特性请求 & 头脑风暴]
That's a very annoying and useless shortcut. When you play a game that utilizes space and you by mistake press alt+space, you are being thrown out of the game and application beneath show it's window properties.

**Comment by chenzhiwei** [comment]
Agree, please remove this annoying shortcut.

**Comment by meiyixiang** [comment]
Hello, we have adopted this suggestion and will gradually realize it in the later version. Thank you for your advice
